app/src/main/java/com/keylesspalace/tusky/ComposeActivity.java

```diff
@@ -164,6 +165,8 @@
     private static final String REPLYING_STATUS_AUTHOR_USERNAME_EXTRA = "replying_author_nickname_extra";
     private static final String REPLYING_STATUS_CONTENT_EXTRA = "replying_status_content";
 
+    private static int maximumTootCharacters = STATUS_CHARACTER_LIMIT;
```

Comment:
I think it doesn't have to be static?.. At least it seems hacky to me

Reply:
I made it static as a simple way of caching it between instances.
If you'd rather, I can make it an instance field instead.

Reply:
I appreciate the idea but maybe we should cache it some other way (e.g. with account data). Shared and mutable are okay by itself and shared and mutable is a recipe for an untestable mess ;)
Let's think about the best way to do it either here or
https://matrix.to/#/#Tusky:matrix.org

Reply:
The caching isn't super important, I just thought it would be a nice side effect, and I didn't think too much about it since there can only be one compose activity at a time (I think?).

But it can just be an instance field; I don't think it's a big deal.

Reply:
Yeah, I just like the idea of caching too but I don't like static fields :sweat_smile: 
Here's a real-life scenario: you change your account and it has another char limit and now we show wrong limit.
It would be cool to cache it while keeping it maintainable but I would approve it even without cache, tbh.
Maybe store the limit in SharedPreferences or make a Room table for instances. We could use some instance info in the future and it would be nice to have it cached.
Uuuugh, @connyduck can you give your opinion on this pls since you will be deciding on it?..

Reply:
If I understand the life cycle correctly, if you change your account, the value would be requeried, and it could only be incorrect if the request failed (which is also true in the normal case). 

But it's not a problem; I'll just change it. ☺